website/README.md

```diff
@@ -2,15 +2,28 @@
 
 This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
+## Prerequisites
+
+To build and test documentation locally, begin by downloading and installing [Node.js](https://nodejs.org/en/download/), and then installing [Yarn](https://classic.yarnpkg.com/en/).
+On Windows, you can install via the npm package manager (npm) which comes bundled with Node.js:
+
+```console
+npm install --global yarn
+```
+
 ## Installation
 
 ```console
 yarn install
+pip install pydoc-markdown
 ```
 
 ## Local Development
 
+Navigate to the website folder and run:
+
 ```console
+pydoc-markdown
```

Comment:
ty for putting these in! anyone else can now recreate docs locally which is definitely nice :)